modules/ossm-configuring-jaeger.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-specifying-jaeger-configuration_{context}"]
+= Specifying Jaeger configuration in the SMCP
+
+You configure Jaeger under the `addons` section of the `ServiceMeshControlPlane` resource. However, there are limitations to what you can configure in the SMCP.
+
+When the SMCP passes configuration information to the {JaegerName} Operator, it triggers one of three deployment strategies: `allInOne`, `production`, or `streaming`.
```

Comment:
Just to highlight, I can't confirm how` {JaegerName}` renders in this module, because the module does not seem to be included in any assemblies.